File: benchmarks/corpus/go/cmdi_exec.go
package main

import (
	"net/http"
	"os/exec"
)

func ping(w http.ResponseWriter, r *http.Request) {
	host := r.URL.Query().Get("host")
	cmd := exec.Command("sh", "-c", "ping -c 1 "+host)
	_ = cmd.Run()
}

File: benchmarks/corpus/javascript/cmdi_exec.js
const express = require("express");
const { exec } = require("child_process");
const app = express();

app.get("/ping", function (req, res) {
  const host = req.query.host;
  exec("ping -c 1 " + host, function (err, stdout) {
    res.send(stdout);
  });
});
